frontend/src/utils/generateRoomName/generateRoomName.ts

```diff
@@ -216,7 +216,6 @@ const animals = [
   'trout',
   'turkey',
   'turtle',
-  'vicuña',
```

Comment:
Note: this is not related to this PR; however, the special symbol in this word introduces some issues with room joining 😅